bridge-api/definitions/shared_module_metadata.yml

```diff
@@ -0,0 +1,55 @@
+type: object
+description: Metadata for shared modules in the shared module library.
+required:
+    - id
+    - name
+    - version
+properties:
+    id:
+        type: string
+        description: Unique ID of the module.
+    licenseRestricted:
+        type: boolean
+        description: True if usage of this module is restricted by license or copyright.
+    moduleType:
+        $ref: ./enums/shared_module_type.yml
+        readOnly: true
```

Comment:
You won't be able to set this in the object if you're constructing it, is that ok?